lightblue-ldap-crud/src/main/java/com/redhat/lightblue/crud/ldap/LdapCRUDController.java

```diff
@@ -533,5 +530,28 @@ private void execute(CRUDOperationContext ctx, ExecutionHandler handler){
         abstract void onSuccess(LDAPResult result);
 
     }
+    
+    @Override
+    public CRUDHealth checkHealth() {
+        boolean isHealthy = true;
+        Map<String, Object> ldapConnnectionsStatus = dbResolver.getLDAPConnectionsStatus();
+        List<String> details = new ArrayList<>(ldapConnnectionsStatus.size());
+
+        for (Map.Entry<String, Object> connectionStatus : ldapConnnectionsStatus.entrySet()) {
+
+            if (connectionStatus.getValue() instanceof LDAPException) {
+                isHealthy = false;
+                details.add(new StringBuilder("LDAPConnection [DatabaseName: ").append(connectionStatus.getKey())
```

Comment:
The Map connectionsStatus specifically uses database name from LdapDataSourceConfiguration as a key. This is that text. If I say LdapServer in the logs, I should provide some bind DN, or IP details of the LdapServer, which is not available at this point in the current implementation.